src/components/CloseButton/index.ts

```diff
@@ -0,0 +1 @@
+export {CloseButton, CloseButtonProps} from './CloseButton';
```

Comment:
If we are going to have `CloseButton`, `Button`, `TextButton`, then Maybe they can all sit under `Button` dir?

Reply:
I think it is a good idea. We can also put all the input in a folder, and so on...
before that, I would like to fix it so that no-one is importing from `/dist/src/...`, because then things like this start to break all the world.